src/meetbot/features/command-help.ts

```diff
@@ -1,21 +1,41 @@
 import { Bot } from '..';
-// import { postToChatJob } from '../pptr-helpers';
-
+import { postToChatJob } from '../pptr-helpers';
 import fs = require('fs');
-// Chat command to send help text for meetbot commands
-const files = fs.readdirSync(__dirname);
+const Comments = require('parse-comments');
+const comments = new Comments();
+
+
+function commandParser(): CommandHelp[] {
```

Comment:
Maybe rename to something like `parseCommandComments` or `commandHelpFromComments?